astro/src/tools/docs/getReleaseNoteRssItems.ts

```diff
@@ -6,7 +6,7 @@ export const getReleaseNoteRssItems = async () => {
   const releaseLines = releaseNotes.body.split("\n");
   const archiveLines = archive.body.split("\n");
   const lines = [...releaseLines, ...archiveLines];
-  const items = lines.map(line => line.match(/ReleaseNoteHeading version="([^"]*)" releaseDate="([^"]*)"/))
+  const items = lines.map(line => line.match(/ReleaseNoteHeading version=['"]([^('|")]*)['"] releaseDate=['"]([^('|")]*)['"]/))
```

Comment:
```suggestion
  const items = lines.map(line => line.match(/ReleaseNoteHeading version=['"]([^'"]*)['"] releaseDate=['"]([^'"]*)['"]/))
```
I think this is what you really want.  Otherwise, you're saying not to match any of the chars `('|")`